specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/Pipeline.json

```diff
@@ -1129,6 +1234,48 @@
         "customRfcReadTableFunctionModule": {
           "type": "object",
           "description": "Specifies the custom RFC function module that will be used to read data from SAP Table. Type: string (or Expression with resultType string)."
+        },
+        "partitionOption": {
+          "description": "The partition mechanism that will be used for SAP table read in parallel.",
+          "type": "string",
+          "enum": [
+            "None",
+            "PartitionOnInt",
+            "PartitionOnCalendarYear",
+            "PartitionOnCalendarMonth",
+            "PartitionOnCalendarDate",
+            "PartitionOnTime"
+          ],
+          "x-ms-enum": {
+            "name": "SapTablePartitionOption",
+            "modelAsString": true
+          }
+        },
+        "partitionSettings": {
+          "description": "The settings that will be leveraged for SAP table source partitioning.",
+          "$ref": "#/definitions/SapTablePartitionSettings"
+        }
+      }
+    },
+    "SapTablePartitionSettings": {
```

Comment:
This too?

Reply:
this is a new setting that is ready for use. I would prefer to keep it in this change.  any specific concerns?

Reply:
by the way, this change has nothing to do with the dataset changes.